python/ray/worker.py

```diff
@@ -1731,6 +1736,7 @@ def init(redis_address=None,
             Store with hugetlbfs support. Requires plasma_directory.
         include_webui: Boolean flag indicating whether to start the web
             UI, which is a Jupyter notebook.
+        driver_id: The id of driver, and it will be ignored in non-driver mode.
```

Comment:
id -> ID

remove the stuff after the `,`, there is no such thing as "non-driver" mode.

Reply:
at least from the perspective of the user and `ray.init`, there is no such thing as "non-driver" mode.